clap-v3-utils/tests/test_try_get_many.rs

```diff
@@ -0,0 +1,58 @@
+use {
```

Comment:
I'd suggest we do not make a separate file with tests that is disconnected from the impl they are testing. If you want to make a separate file with tests, suggest making sure its name is more descriptive